client/src/constants/routes.js

```diff
@@ -1,6 +1,8 @@
 export default Object.freeze({
   // Public routes
   Login: '/login',
+  Admin: '/admin',
```

Comment:
Tiny nitpick but would love to move this to its own block commented for private routes vs. being in with the public ones here :)